pkg/controller/logcollector/logcollector_controller.go

```diff
@@ -429,6 +432,22 @@ func (r *ReconcileLogCollector) Reconcile(ctx context.Context, request reconcile
 		}
 	}
 
+	var useUserCA bool
+	if instance.Spec.AdditionalStores != nil {
+		if instance.Spec.AdditionalStores.Syslog != nil {
+			syslogCert, err := getSysLogCertificate(r.client)
```

Comment:
Shouldn't this only happen if encryption == TLS?

Reply:
yes.. Updated